### PR TITLE
importWaterML2 to xml2

### PR DESCRIPTION
**Pull request by wdwatkins**
Just importing the basic info right now - time, value, qualifier, and identifier.  Will be fleshed out for NGWMN soon

### CODE REVIEW

File: R/importWaterML1.r [hunk 1]
```diff
@@ -97,356 +95,194 @@
...
-            
-            numChar <- nchar(datetime)
-            
-            datetime <- parse_date_time(datetime, c("%Y","%Y-%m-%d","%Y-%m-%dT%H:%M",
-                                                    "%Y-%m-%dT%H:%M:%S","%Y-%m-%dT%H:%M:%OS",
-                                                    "%Y-%m-%dT%H:%M:%OS%z"), exact = TRUE)
-            
-            if(any(numChar < 20) & any(numChar > 16)){
-              
-              offsetLibrary <- data.frame(offset=c(5, 4, 6, 5, 7, 6, 8, 7, 9, 8, 10, 10, 0),
```
Comment (ldecicco-USGS) [inline R/importWaterML1.r]: @wdwatkins ...are you getting around this conversion somewhere else?

**Comment by ldecicco-USGS** [comment]
One thing that's different is that the "production" version of dataRetrieval (that is on CRAN) changes the "no value" values to NA. So that the user doesn't need to check if the value == -999999. 

So, you'll want to get the "no value" value here:
https://github.com/USGS-R/dataRetrieval/blob/master/R/importWaterML1.r#L161
Then change it to NA something like:
https://github.com/USGS-R/dataRetrieval/blob/master/R/importWaterML1.r#L422

**Comment by ldecicco-USGS** [comment]
For the siteInfo attribute, `dec_lat_va` and `dec_lon_va` need to be numeric.

**Comment by ldecicco-USGS** [comment]
For the variableInfo attribute, if you convert the "no value" value to NA, then change the noDataValue to NA as well.

**Comment by ldecicco-USGS** [comment]
Column order is different. Most specifically the tz_cd column should go to the end. In the older versions, the values are lumped together, and the cd's are lumped together. I prefer them to be lumped by parameter code (as your new version has)...but we have to be prepared for people's scripts breaking. I'd say leave your order as is, but move the tz_cd to the far right.

For example:

```
bBoxEx <- readNWISdata(bBox=c(-83,36.5,-81,38.5), parameterCd="00010")
```

The CRAN version is:
```
[1] "agency_cd"        "site_no"          "dateTime"         "X_00010_00001"   
 [5] "X_00010_00002"    "X_00010_00003"    "X_00010_00011"    "X_00010_00008"   
 [9] "X_00010_00001_cd" "X_00010_00002_cd" "X_00010_00003_cd" "X_00010_00011_cd"
[13] "X_00010_00008_cd" "tz_cd"   
```
This version is:
```
[1] "agency_cd"        "site_no"          "dateTime"         "X_00010_00001"   
 [5] "X_00010_00001_cd" "tz_cd"            "X_00010_00008"    "X_00010_00008_cd"
 [9] "X_00010_00003"    "X_00010_00003_cd" "X_00010_00002"    "X_00010_00002_cd"
[13] "X_00010_00011"    "X_00010_00011_cd"
```

**Comment by ldecicco-USGS** [comment]
Let's change the names in the statisticInfo attribute to remain backwards compatible:
```
[1] "statisticName" "statisticCd"  
```
instead of:
```
[1] "statCd"  "stat_nm"
```

**Comment by ldecicco-USGS** [comment]
Moving to #268

**Comment by wdwatkins** [comment]
restarting travis

**Comment by wdwatkins** [comment]
Try again

**Pull request merged**